src/els_dt_document.erl

```diff
@@ -72,12 +72,7 @@ name() -> ?MODULE.
 
 -spec opts() -> proplists:proplist().
 opts() ->
-  [ {attributes        , record_info(fields, els_dt_document)}
-  , {disc_copies       , [node()]}
-  , {index             , []}
-  , {type              , set}
-  , {storage_properties, [{ets, []}]}
-  ].
+  [].
```

Comment:
Is `set` the default?  Should we be explicit?

Reply:
Yes, `set` is the default type. We can be explicit, will do it as a separate PR (I want to change something else).